.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: test
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 14, 16]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - run: npm install --ignore-scripts
+      - run: npm run build
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm install --ignore-scripts
+      - run: npm run test
```

Comment:
Build fails in node 12, but this is fine.
We only support older node versions externally, not persé for building the plugin itself.

By switching node version and running install again we can build in node 16 and then test in node X